### PR TITLE
Oddbee: Add anchor icons, fix nav, fix misc styles

### PR DESCRIPTION
**Pull request by y-kovaleva**
Small functionality and style fixes:

- Add page title before main doc content - helps with mobile nav
- Add an anchor tag inside each h2 and add a class to it - creates automatic anchor links for each h2
- Update footer to match marketing site + learning centre
- Small miscellaneous style updates

### CODE REVIEW

File: assets/img/icon-anchor.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg width="28" height="28" viewBox="0 0 28 28" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (Moux) [inline assets/img/icon-anchor.svg:1]: This file needs to be moved to assets/img/icons ,or the anchor icon won't be shown.

**Review by Moux — CHANGES_REQUESTED** [review]
Just move the location of the anchor icon file, and then its good.

**Comment by Moux** [comment]
Title is showing up twice in the pages, but it's probably better to make this change in the markdown, since the content is one block.

**Comment by orospakr** [comment]
hey @y-kovaleva and @Moux 

This PR was set to the wrong base (dev branch), even though clearly the work was (correctly) done off the master brach. That was why this PR was showing a bunch of commits from Biso and myself, causing the diff to show a bunch of unrelated (and already completed) changes.

I just tweaked the PR's base setting and issue is now resolved :)

**Review by Moux — APPROVED** [review]

**Review by Moux — APPROVED** [review]

**Pull request merged**